app/models/feed.rb

```diff
@@ -89,7 +89,7 @@ def feed_episodes
     feed_max = display_episodes_count.to_i
 
     filtered_episodes.each do |ep|
-      include_in_feed << ep if ep.media_ready?
+      include_in_feed << ep if ep.include_in_feed?
```

Comment:
Seemed incorrect to have been calling `media_ready?` here.  Was missing some of the logic `include_in_feed?` had.